<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Thiago Nascimento</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QAgentes Lógicos</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>resources/aspirador.png</normaloff>resources/aspirador.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>320</width>
      <height>320</height>
     </rect>
    </property>
    <property name="resizeAnchor">
     <enum>QGraphicsView::AnchorViewCenter</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Agente</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="labelAgentePosX">
        <property name="text">
         <string>PosX: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelAgentePosY">
        <property name="text">
         <string>PosY:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>339</x>
      <y>109</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Informações</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>141</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="labelInfoPassos">
        <property name="toolTip">
         <string>Total de passos dados pelo agente</string>
        </property>
        <property name="text">
         <string>Passos:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelInfoSujeira">
        <property name="toolTip">
         <string>Número de salas limpa</string>
        </property>
        <property name="whatsThis">
         <string>oi</string>
        </property>
        <property name="text">
         <string>Sujeira:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelInfoPontos">
        <property name="toolTip">
         <string>Pontuação Geral do agente</string>
        </property>
        <property name="text">
         <string>Pontos:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>230</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Bateria</string>
    </property>
    <widget class="QProgressBar" name="pbBateriaAgente">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>518</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionSobre"/>
    <addaction name="actionSobre_Qt"/>
   </widget>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <widget class="QMenu" name="menuNovo">
     <property name="title">
      <string>Novo</string>
     </property>
     <addaction name="actionNovoAmbiente"/>
     <addaction name="separator"/>
     <addaction name="actionNovoAgenteSimples"/>
     <addaction name="actionNovoAgenteMemoriaAnterior"/>
     <addaction name="actionNovoAgenteMemoria"/>
     <addaction name="actionNovoAgenteGuloso"/>
     <addaction name="actionNovoAgenteInteligente"/>
    </widget>
    <addaction name="menuNovo"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuTestes">
    <property name="title">
     <string>Testes</string>
    </property>
    <addaction name="actionExecutarTestes"/>
   </widget>
   <widget class="QMenu" name="menuExecutar">
    <property name="title">
     <string>Ação</string>
    </property>
    <addaction name="actionExecutar"/>
    <addaction name="actionParar"/>
    <addaction name="separator"/>
    <addaction name="actionResetar"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuExecutar"/>
   <addaction name="menuTestes"/>
   <addaction name="menuAjuda"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionSobre_Qt">
   <property name="text">
    <string>Sobre Qt</string>
   </property>
  </action>
  <action name="actionNovoAmbiente">
   <property name="text">
    <string>Ambiente</string>
   </property>
  </action>
  <action name="actionNovoAgenteSimples">
   <property name="text">
    <string>Agente Simples</string>
   </property>
  </action>
  <action name="actionExecutar">
   <property name="text">
    <string>Executar</string>
   </property>
   <property name="statusTip">
    <string>Disparar agente!</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionParar">
   <property name="text">
    <string>Parar</string>
   </property>
  </action>
  <action name="actionResetar">
   <property name="text">
    <string>Resetar</string>
   </property>
  </action>
  <action name="actionNovoAgenteMemoriaAnterior">
   <property name="text">
    <string>Agente Com Memória Anterior</string>
   </property>
  </action>
  <action name="actionNovoAgenteMemoria">
   <property name="text">
    <string>Agente com Memória Completa</string>
   </property>
  </action>
  <action name="actionNovoAgenteGuloso">
   <property name="text">
    <string>Agente Guloso</string>
   </property>
  </action>
  <action name="actionNovoAgenteInteligente">
   <property name="text">
    <string>Agente Inteligente</string>
   </property>
  </action>
  <action name="actionExecutarTestes">
   <property name="text">
    <string>Executar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionSair</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>signal1()</signal>
  <slot>aboutQt()</slot>
 </slots>
</ui>
